src/helm/benchmark/metrics/basic_metrics.py

```diff
@@ -268,6 +268,16 @@ def chinese_bleu_1(gold: str, pred: str) -> float:
     return sentence_bleu([[i for i in gold]], [i for i in pred], weights=(1, 0, 0, 0))
 
 
+def get_chinese_rouge_function(rouge_type: str) -> Callable[[str, str], float]:
+    class CharTokenizer:
+        def tokenize(self, text: str) -> List[str]:
+            return [char for char in text]
+
+    char_tokenizer = CharTokenizer()
+    scorer = rouge_scorer.RougeScorer([rouge_type], use_stemmer=True, tokenizer=char_tokenizer)
```

Comment:
Which `rouge_score` are you using? Is it the multilingual version or English-only version?

Reply:
I think we should use the English-only version as it is hard to add the dependency of the multilingual version. A way to resolve the word segmentation issue of English-only ROURGE is that we tokenize Chinese text and use space to concatenate all words again before passing it into the ROUGE score function.